lib/json_api_filter/field_filters/sorter.rb

```diff
@@ -0,0 +1,16 @@
+module JsonApiFilter
+  module FieldFilters
+    class Sorter < Base
+  
+      # @return [ActiveRecord_Relation]
+      def predicate
+        return nil if values["by"].nil?
+        result = scope
+        result = result.order(values["by"])
+        result = result.reverse_order if values["desc"] == "true"
+        result
+      end
+    
```

Comment:
I would extract this condition

```suggestion
            scope.order(values["by"] => order)
      end
      
      def order
        ActiveModel::Type::Boolean.new.cast(values["desc"]) ? :desc : :asc
      end
    
```